### PR TITLE
AST-1003 - Option to display selected font-families in dropdown

### PR DESCRIPTION
**Pull request by GauravKhupse**
### Description
It will be a global option in settings
User will select a few font-family as per his requirement
Based on the user's selection will display the selected font families in the customizer dropdown.

### Types of changes

 New feature (non-breaking change which adds functionality) 

### Checklist:
- [x] My code is tested
- [x] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: inc/assets/js/astra-admin-menu-settings.js [hunk 1]
```diff
@@ -288,10 +289,47 @@
...
+				type: "POST",
+				dataType: 'json',
+				url: astra.ajaxUrl,
+				data: {
+					action: 'global_font_family',
+					fontfamily: fontFamilySelected,
+					fontfamilynonce: astra.ajax_nonce,
+				},
+				success: function (output) {
+				   console.log(output);
```
Comment (imnavanath) [inline inc/assets/js/astra-admin-menu-settings.js:328]: log can be remove.

File: inc/assets/js/astra-admin-menu-settings.js [hunk 2]
```diff
@@ -288,10 +289,47 @@
...
 	 * Initialize AstraThemeAdmin
 	 */
 	$(function(){
+		
+
+		let fontFamilySelected; 
+	
+		$('.ast-global-font-family').select2();
+		$('.ast-global-font-family').select2({
+			placeholder: 'This is my placeholder',
```
Comment (imnavanath) [inline inc/assets/js/astra-admin-menu-settings.js:304]: check placeholder is needed & if yes then check their string.

File: inc/assets/js/astra-admin-menu-settings.js [hunk 3]
```diff
@@ -288,10 +289,47 @@
...
 	/**
 	 * Initialize AstraThemeAdmin
 	 */
 	$(function(){
+		
+
+		let fontFamilySelected; 
+	
+		$('.ast-global-font-family').select2();
+		$('.ast-global-font-family').select2({
```
Comment (imnavanath) [inline inc/assets/js/astra-admin-menu-settings.js:303]: I don't think, two declarations are needed??

File: inc/assets/js/astra-admin-menu-settings.js (hunk 1 above)
Comment (GauravKhupse) [reply inc/assets/js/astra-admin-menu-settings.js]: Changes done!

File: inc/assets/js/astra-admin-menu-settings.js (hunk 3 above)
Comment (GauravKhupse) [reply inc/assets/js/astra-admin-menu-settings.js]: Changes done!

**Pull request closed without merging**